src/NServiceBus.Core.Tests/Sagas/ComponentTests/PersistenceTestsConfiguration.cs

```diff
@@ -0,0 +1,42 @@
+﻿// ReSharper disable AutoPropertyCanBeMadeGetOnly.Global
+namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using Extensibility;
+    using Sagas;
+
+    public partial class PersistenceTestsConfiguration : IPersistenceTestsConfiguration
+    {
+        public Func<ContextBag> GetContextBagForTimeoutPersister { get; set; } = () => new ContextBag();
+        public Func<ContextBag> GetContextBagForSagaStorage { get; set; } = () => new ContextBag();
+        public Func<ContextBag> GetContextBagForOutbox { get; set; } = () => new ContextBag();
+        public Func<ContextBag> GetContextBagForSubscriptions { get; set; } = () => new ContextBag();
+
+        public SagaMetadataCollection SagaMetadataCollection
+        {
+            get{
+                if (sagaMetadataCollection == null)
+                {
+                    sagaMetadataCollection = new SagaMetadataCollection();
+                    sagaMetadataCollection.Initialize(new[]
+                    {
+                        typeof(AnotherSagaWithCorrelatedProperty),
+                        typeof(AnotherSagaWithoutCorrelationProperty),
```

Comment:
`AnotherSagaWithoutCorrelationProperty` == `AnotherSagaWithCustomFincer` ?
@danielmarbach mentioned standardizing on the naming and use either Found by Correlation Property or Custom Finder.